src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -476,10 +476,15 @@ bool OutputStateMachineEngine::ActionCsiDispatch(const VTID id, const VTParamete
         success = _dispatch->CursorPosition(parameters.at(0), parameters.at(1));
         TermTelemetry::Instance().Log(TermTelemetry::Codes::CUP);
         break;
-    case CsiActionCodes::DECSTBM_SetScrollingRegion:
+    case CsiActionCodes::DECSTBM_SetTopBottomMargins:
         success = _dispatch->SetTopBottomScrollingMargins(parameters.at(0).value_or(0), parameters.at(1).value_or(0));
         TermTelemetry::Instance().Log(TermTelemetry::Codes::DECSTBM);
         break;
+    case CsiActionCodes::DECSLRM_SetLeftRightMargins:
+        // Note that this can also be ANSISYSSC, depending on the state of DECLRMM.

```

Comment:
Yeah, if you look at the `SetLeftRightScrollingMargins` implementation in `AdaptDispatch`, there's a test there that forwards this sequence to `CursorSaveState` if the left-right margin mode hasn't been activated yet. So by default it can still be used to save the cursor state. It's only once `DECSLRMM` has been enabled that it'll be used to control the margins instead.